### PR TITLE
Add FAQ which can be managed through bot commands

### PR DESCRIPTION
**Pull request by NicEastvillage**
This PR adds the ability to manage an FAQ channel through commands. This allows us to have a FAQ channel with a streamlined format and where everyone can adjust/update/add if needed. Rearranging questions is also easy. The following commands are added:
* `faq_channel <#channel_id>` : Sets the FAQ channel. Will probably only be used once.
* `add_faq <question> <answer>` : Appends a question to the list of FAQs
* `edit_faq <qid> <question> <answer>` : Updates question `qid` to the given question and answer
* `swap_faqs <qid1> <qid2>` : Swaps question `qid1` and `qid2`
* `del_faq <qid>` : Removes question `qid`
* `refresh_faq` : Makes the bot resend the FAQ in case something went wrong or formatting was updated 

This is how it will look like:
billede

### CODE REVIEW

**Comment by VirxEC** [comment]
This looks cool - even if we decide not to have an FAQ, having this tool in the bot will still be nice.

File: RLBotDiscordBot/cogs/faq.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        await self.refresh(ctx)
+
+    async def refresh(self, ctx):
+
+        await self.remove_old_faq_messages(ctx)
+
+        # Validate that faq channel exists
+        faq_channel_id = self.bot.settings.get('Faq_channel')
+        if faq_channel_id is None:
+            print("hi1")
```
Comment (skyborgff) [inline RLBotDiscordBot/cogs/faq.py:132]: Remove debug prints :P

File: RLBotDiscordBot/cogs/faq.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            await ctx.send('FAQ channel is not set. Use `!faq_channel <#channel_id>` to set it')
+            return
+        faq_channel = ctx.guild.get_channel(faq_channel_id)
+        if faq_channel is None:
+            print("hi2")
```
Comment (skyborgff) [inline RLBotDiscordBot/cogs/faq.py:137]: Remove debug prints :P V2

**Review by skyborgff — CHANGES_REQUESTED** [review]
Also remove the binary RLBotDiscordBot/cogs/__pycache__/admin.cpython-37.pyc

I'll test it later and approve after the changes

**Review by skyborgff — APPROVED** [review]
Looks good :)

**Pull request merged**